SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp

```diff
@@ -449,3 +450,23 @@ bool USpatialGDKEditorSettings::IsDeploymentConfigurationValid() const
 
 	return bValid;
 }
+
+FString USpatialGDKEditorSettings::GetCookAndGenerateSchemaTargetPlatform() const
+{
+	if (!CookAndGeneratePlatform.IsEmpty())
+	{
+		return CookAndGeneratePlatform;
+	}
+
+	const FString PreferredPlatformForCookAndGenerateSchema(TEXT("Linux"));
```

Comment:
Hmmm does this mean if I have developer only content which isn't included in Linux builds that I won't get schema generated for my editor only content? Let's say if I was Win64 or Mac. I think the default should be the editor platform and the fallback be Linux perhaps. Unless I'm wrong about the above.

Reply:
This behaviour is trying to reproduce what is advised in CookAndGenerateSchemaCommandlet.h .
And since the server will be Linux eventually, I think it is a reasonable requirement that everything you want to replicate have to be Linux-available content.

Reply:
Yeah but this is cooking for the editor. In the editor. It should be the editor platform default. The commandlet uses Linux by default because that's all that's required for CI no? @jessicafalk for context.